src/main/java/net/sf/jabref/gui/fieldeditors/JTextAreaWithHighlighting.java

```diff
@@ -77,39 +77,37 @@ public void undoableEditHappened(UndoableEditEvent evt) {
         });
 
         // Create an undo action and add it to the text component
-        getActionMap().put("Undo",
-                new AbstractAction("Undo") {
-
-                    @Override
-                    public void actionPerformed(ActionEvent evt) {
-                        try {
-                            if (undo.canUndo()) {
-                                undo.undo();
-                            }
-                        } catch (CannotUndoException ignored) {
-                    // Ignored
-                        }
+        getActionMap().put("Undo", new AbstractAction(Actions.UNDO) {
```

Comment:
I *think* that it is just a arbitrary name, but it may be better to not change it... This naming vs Action.CONSTANT should be clarified somehow. Or maybe it is clear to other people than me...

I'll add the ChangeLog entry and also remove the other stuff discussed.